packages/eslint-plugin/lib/rules/utils/baseline.js

```diff
@@ -0,0 +1,558 @@
+/**
+ * This file is auto-generated. (yarn run baseline)
+ */
+const BASELINE_HIGH = 10;
+const BASELINE_LOW = 5;
+const BASELINE_FALSE = 0;
+
+const elements = new Map([
+  ["a", "10:2015"],
+  ["a.attributionsourceid", "0:"],
+  ["a.attributionsrc", "0:"],
+  ["a.download", "10:2019"],
+  ["a.href", "10:2015"],
+  ["a.href.href_sms", "0:"],
```

Comment:
You can ignore any keys like this containing an underscore. Those don't correspond to the element/attribute itself, they're more like implementation details.

Reply:
@rviscomi Thanks! I fixed it. https://github.com/yeonjuan/html-eslint/pull/309/commits/31ddf3d65adadd9727ba768ce5ada095b14fffee